packages/theme/webpack.config.js

```diff
@@ -26,62 +29,59 @@ module.exports = () => {
 				},
 				{
 					test: /bootstrap\.scss$/,
-					loader: ExtractTextPlugin.extract({
-						fallback: 'style-loader',
-						use: [
-							{
-								loader: 'css-loader',
-								options: {
-									importLoaders: 3,
-									minimize: true,
-									sourceMap: false,
-								},
+					use: [
+						{
+							loader: isDev ? 'style-loader' : MiniCssExtractPlugin.loader,
+						},
+						{
+							loader: 'css-loader',
+							options: {
+								importLoaders: 3,
+								sourceMap: true,
```

Comment:
Is it intentional?
```suggestion
								sourceMap: isDev,
```

Reply:
Yes 
👉  https://unpkg.com/browse/@talend/bootstrap-theme@6.16.0/dist/
source map is already released